dist/pages/video-download/video-download.view.html

```diff
@@ -0,0 +1,30 @@
+<div ng-controller="VideoDownloadController as VideoDownload" class="download-page">
+	<article>
+		<video controls ng-src="{{::VideoDownload.video.data.video}}" ng-attr-poster="{{::VideoDownload.video.data.video}}" preload="metadata" controlsList="nodownload" class="video-download-page"></video>
```

Comment:
I've set the `preload="metadata"` to get the canvas its correct size.... If having the player without size doesn't matter, then we could set this property to `none`